packages/odyssey-react/src/components/Radio/RadioGroup/index.tsx

```diff
@@ -71,47 +61,48 @@ export interface Props
  * Radios appear as a ring shaped UI accompanied by a caption that allows
  * the user to choose only one option at a time.
  */
-const RadioGroup = forwardRef<HTMLFieldSetElement, Props>((props, ref) => {
```

Comment:
Discussed this removal in #odyssey-core. Short version: this would require Field to have a polymorphic ref based on the associated `as` prop. Because this isn't necessary for any functionality in the RadioGroup container, we've nixed this until we have a standard pattern.